waggle-dance-integration-tests/src/main/java/com/hotels/bdp/waggledance/WaggleDanceRunner.java

```diff
@@ -38,6 +36,7 @@
 import org.apache.commons.vfs2.FileSystemException;
 import org.apache.commons.vfs2.FileSystemManager;
 import org.apache.commons.vfs2.VFS;
+import org.apache.logging.log4j.util.Strings;
```

Comment:
I don't think we should be using this `Strings` util class but rather one from `guava` or `apache-commons`